ucsb-cs156-f24/team02-f24-00
# Pc create commit page

Closes: #95 

Merge after:  #99 and #112 (this depends on **both**) 

In this PR, we replace the placeholder Commit Create Page with a working Create page for commits.

Storybook: https://67217d09054c99a597a2d9ac-kaypipvrgc.chromatic.com/?path=/docs/pages-commits-commitscreatepage--docs

# To test: 

1. login to this dokku instance: https://team02-dev-pconrad.dokku-00.cs.ucsb.edu/
2. navigate to the create page for Commits
3. Fill in the fields
4. Click Create
5. Check the /api/commits/all endpoint with Swagger to see if the new record is in the database
https://team02-dev-pconrad.dokku-00.cs.ucsb.edu/
